ee_plugin/processing/add_image_collection.py

```diff
@@ -447,17 +446,10 @@ def processAlgorithm(self, parameters, context, feedback):
         if extent and isinstance(extent, str):
             # Parse extent from string format: "xmin,ymin,xmax,ymax [CRS]"
             try:
-                coords = (
-                    extent.split(" [")[0].replace(" : ", ",").replace(":", ",")
-                )  # Normalize separator
-                xmin, ymin, xmax, ymax = map(float, coords.split(","))
-                extent = QgsRectangle(xmin, ymin, xmax, ymax)
-                min_lon = extent.xMinimum()
-                max_lon = extent.xMaximum()
-                min_lat = extent.yMinimum()
-                max_lat = extent.yMaximum()
+                coords = extent.split(" [")[0]
+                min_lon, max_lon, min_lat, max_lat = map(float, coords.split(","))
```

Comment:
So we were mistaken about the stored format?  If so, we should likely update the comment on 447